### ISSUE TITLE
Add forwarding for other agents running in the system, like gpg-agent

### ISSUE THREAD

**Issue by ntrippar** [ntrippar/sekey#2]
Implement forwarding of messages to check if in other agents in the systems exist the key that we need.

**Comment by balupton** [comment]
This would be a great solution to the "get locked out of servers which only have one ssh key" issue raised in https://github.com/ntrippar/sekey/issues/11

Which is actually an issue I face with certain servers I use. So holding off adoption of sekey until this feature is done.

Looking forward to using this though, seems wicked!

**Comment by balupton** [comment]
I've also seen that https://krypt.co are considering similar functionality too over at this issue
https://github.com/kryptco/kr/issues/149

Perhaps the solution here is to make an additional ssh agent, which is a generic one, that just allows one to specify an ordered list of other ssh agents to defer to. So that new generic ssh agent, would have a ordered list that says, first try the local ssh agent, then try sekey, then try kryptco.

**Comment by ntrippar** [comment]
@balupton yes totally, I also use a yubikey for some other work reasons, the code for the forwarding will be push next week!. its almost done.

**Comment by ntrippar** [comment]
it should work, im using in that way @nimish, as seen in other tickets im building a new version to debug, i will keep this open and upload the new version. (been with a lot of work so I didnt have time to build it). i will do tomorrow.

**Comment by webframp** [comment]
Looking forward to it!

**Comment by martinpaljak** [comment]
There are other “frontend” ssh agent proxies out there. I would suggest using them for fancy setups and leave sekey for a rock solid backend implementation with touchid only.

**Comment by nikkikom** [comment]
> There are other “frontend” ssh agent proxies out there. 

Would you please name few?

**Comment by martinpaljak** [comment]
Like https://github.com/tiwe-de/ssh-agent-filter. 

But following the unix philosophy, it would be great if sekey worked as a standalone agent accessing the SE, and other routing/filtering/selection would happen in a separate agent.
